tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -150,60 +159,113 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     };
 
     fluid.tests.tooltipModuleSource = function (pager) {
-        var pageLinksTop = $("a", pager.pagerBar.locate("pageLinks"));
-        var pageLinksBottom = $("a", pager["pagerBar-1"].locate("pageLinks"));
-
-        var tooltipContents = [
-            [
-                {nodeName: "b", nodeText: "bird"},
-                {nodeName: "b", nodeText: "fish"}
-            ]
-        ];
-
         var sequence = [];
 
-        var assertVisibleTips = function (message, targetIds) {
-            sequence.push({
-                event: "{trackTooltips}.events.notifyFocusChange",
-                listener: function () {
-                    fluid.tests.tooltip.assertVisible(message, pager, targetIds, null, function (tooltip) {
-                        jqUnit.assertNode("The contents of the tooltip should be set", tooltipContents[0], $("b", tooltip));
-                    });
+        /* Change the page size and verify whether the tooltips are synced accordingly */
+
+        fluid.tests.setPageSize(pager, sequence, 2);
+        fluid.tests.verifyToolTips(pager, sequence, 2, [
+            [ {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "cat"} ],
+            [ {nodeName: "b", nodeText: "bird"}, {nodeName: "b", nodeText: "fish"} ],
+            [ {nodeName: "b", nodeText: "camel"}, {nodeName: "b", nodeText: "dragon"} ],
+            [ {nodeName: "b", nodeText: "ant"}, {nodeName: "b", nodeText: "ant"} ]
+        ]);
+
+        fluid.tests.setPageSize(pager, sequence, 3);
+        fluid.tests.verifyToolTips(pager, sequence, 3, [
+            [ {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "bird"} ],
+            [ {nodeName: "b", nodeText: "fish"}, {nodeName: "b", nodeText: "dragon"} ],
+            [ {nodeName: "b", nodeText: "ant"}, {nodeName: "b", nodeText: "ant"} ]
+        ]);
+
+        fluid.tests.setPageSize(pager, sequence, 4);
+        fluid.tests.verifyToolTips(pager, sequence, 4, [
+            [ {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "fish"} ],
+            [ {nodeName: "b", nodeText: "camel"}, {nodeName: "b", nodeText: "ant"} ]
+        ]);
```

Comment:
actually i think you'd just modify `fluid.tests.tooltipModuleSource` to take in the above structure as an argument.